modules/activiti-engine/src/test/resources/org/activiti/engine/test/bpmn/event/signal/ThrowSignalEventSessionManagerTest.testSignalRaceConditionsAfterParallelGateway.bpmn20.xml

```diff
@@ -0,0 +1,276 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<definitions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:activiti="http://activiti.org/bpmn" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:modeler="http://activiti.com/modeler" xmlns:omgdc="http://www.omg.org/spec/DD/20100524/DC" xmlns:omgdi="http://www.omg.org/spec/DD/20100524/DI" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xsi:schemaLocation="http://www.omg.org/spec/BPMN/20100524/MODEL BPMN20.xsd" modeler:version="1.0en" modeler:exportDateTime="20171201010336345" modeler:modelId="33" modeler:modelVersion="7" modeler:modelLastUpdated="1512090194573" exporter="Camunda Modeler" exporterVersion="1.11.2" targetNamespace="http://bpmn.io/schema/bpmn">
+  <signal id="signal1" activiti:scope="processInstance" name="Signal1"/>
+  <process id="testSignalRaceConditionsAfterParallelGateway" name="Racing Event Signals After Parallel Gateway" isExecutable="true">
+    <startEvent id="StartEvent_1">
+      <extensionElements>
+        <modeler:editor-resource-id>StartEvent_1</modeler:editor-resource-id>
+      </extensionElements>
+      <outgoing>sid-0A4A04CE-9653-4B46-B81C-FE8BC3B2E2AA</outgoing>
+    </startEvent>
+    <parallelGateway id="ExclusiveGateway_1">

```

Comment:
I'm sorry for pointing at a minor error, but why is it giving the label 'ExclusiveGateway' even though it is a parallelGateway?

Reply:
Fixed